packages/ui/src/components/Accordion/Accordion.stories.tsx

```diff
@@ -0,0 +1,58 @@
+import "../../../public/tab-theme.css";
+import { Meta, StoryFn } from "@storybook/react";
+import React from "react";
+import SubPane from "../SubPane/SubPane";
+
+import { Accordion } from "./index";
+
+type AccordionType = typeof Accordion;
+type AccordionProperties = Meta<AccordionType>;
+
+const AccordionStory: AccordionProperties = {
+  title: "UI/Accordion",
+  component: Accordion,
+};
+
+const Template: StoryFn<AccordionType> = (arguments_) => (
+  <Accordion {...arguments_}>
+    <SubPane icon="icon1.jpg" title="Topic 1">
+      Pane 1 content
+    </SubPane>
+    <SubPane icon="icon2.jpg" title="Topic 2">
+      Pane 2content
+    </SubPane>
+  </Accordion>
+);
+
+export const ActiveIndex: AccordionProperties = Template.bind({});
+export const HorizontalDirection: AccordionProperties = Template.bind({});
+export const VerticalDirection: AccordionProperties = Template.bind({});
+export const CanSelfCollapse: AccordionProperties = Template.bind({});
+export const ActiveIcon: AccordionProperties = Template.bind({});
+export const InactiveIcon: AccordionProperties = Template.bind({});
+
+ActiveIndex.args = {
+  defaultActiveIndex: 10,
```

Comment:
defaultActiveIndex is the index of accordion pane that is open by default. Since we have only two item make it either 0 or 1.